_posts/2024-08-23-announcing-ballerina-2201.10.0-swan-lake-update-10.md

```diff
@@ -0,0 +1,54 @@
+
+The OpenAPI package allows developers to automatically generate and embed the OpenAPI specification for a Ballerina service via a resource. This has been improved by exposing another resource to view the HTML-rendered version of the OAS. This feature allows you to introspect the service in a more intuitive and readable format. For more information, see the section on [OAS resources](https://ballerina.io/spec/http/#236-openapi-specification-resources).
+
+<img alt="Service contract type" src="/images/service_contract_type.png">
+
+## New Connectors
+
+In this release, we've made substantial improvements to the functionality and usability of some connectors. Here are the key highlights:
+
+- As part of the connector revamp initiative, several connectors have been updated significantly. The newly released [`discord`](https://central.ballerina.io/ballerinax/discord/latest) and [`dayforce`](https://central.ballerina.io/ballerinax/dayforce/latest) connectors support interaction with their latest REST API versions. Additionally, the [`asb`](https://central.ballerina.io/ballerinax/asb/latest), [`slack`](https://central.ballerina.io/ballerinax/slack/latest), [`stripe`](https://central.ballerina.io/ballerinax/stripe/latest), and [`twitter (X)`](https://central.ballerina.io/ballerinax/twitter/latest) connectors have undergone major updates, featuring significant API and functionality changes, along with enhanced documentation, making it easier to develop complex applications.
+
+- For SAP integrations, the [`sap`](https://central.ballerina.io/ballerinax/sap/latest) connector has been upgraded to support the invocation of any OData endpoint in S/4HANA and now offers automatic CSRF token authentication, enhancing both security and usability. Similarly, the [`sap.jco`](https://central.ballerina.io/ballerinax/sap.jco/latest) connector now supports the invocation of RFC modules with better type binding and allows the transmission and reception of iDocs. Additionally, the [`sap.s4hana.sales`](https://central.ballerina.io/search?q=sap.s4hana.sales&sort=relevance%2CDESC&page=1&m=packages) connectors provide a high-level interface for Sales and Distribution (SD) APIs in S/4HANA, enabling streamlined sales order management, delivery processing, and billing operations.
+
+- The [`salesforce`](https://central.ballerina.io/ballerinax/salesforce/latest) connector now includes a listener with streaming API support for real-time data synchronization and event-driven workflows. Additionally, the introduction of the `salesforce.types` submodule enables the use of base types, and the reorganization of Salesforce clients enhances usability.
+
+## AI features 
+
+Ballerina Swan Lake Update 10 contains an experimental test generator for Ballerina Copilot and significant improvements to the existing AI data mapper as well. These experimental features will greatly enhance developer productivity and streamline the integration development lifecycle.
+- Test Generator
+<br>
```

Comment:
Why just here?

Reply:
Here we have a subsection title and the section content. In other places we don't.